test/PluginGeneratorTest.ts

```diff
@@ -8,15 +8,18 @@ describe('generator-tinymce:plugin', () => {
       .withPrompts({
         packageName: 'cool-plugin',
         type: 'plugin',
-        description: 'my cool plugin'
+        description: 'my cool plugin',
+        author: 'Tiny Technologies, Inc',
+        email: 'is-accounts@ephox.com',
```

Comment:
I know this was in the old code, but I do wonder if we should have this email address in something public? Do you think it'd be worth switching it to a more obviously-dummy email?

Reply:
It's public already, as it shows as the email for all our NPM packages so I don't see any issue. The email will also be in git history, so it's already exposed (this just moved from the hard coded value to tests).

```
$ npm info @tinymce/oxide

@tinymce/oxide@1.6.1 | LGPL-2.1 | deps: none | versions: 108
TinyMCE 5 Oxide skin

keywords: TinyMCE

dist
.tarball: https://registry.npmjs.org/@tinymce/oxide/-/oxide-1.6.1.tgz
.shasum: 6858332a0cae305550bfad4265b076666f7cccad
.integrity: sha512-yc6KunlOod+OhCnjK1RA6CPUTaO4kPfe1yH93iAhYkHBmajeU3BjjekQnTjGHoLgHSNKJ1v0gPomdzVSiNb6Ag==
.unpackedSize: 1.3 MB

maintainers:
- ephox <npm-publish@tiny.cloud>
- tinymce <is-accounts@ephox.com>

dist-tags:
latest: 1.6.1  

published a month ago by ephox <npm-publish@tiny.cloud>
```

Reply:
Changed it to `no-reply@tiny.cloud` anyways.